recipes/gudhi/meta.yaml

```diff
@@ -0,0 +1,57 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set version = "2.3.0" %}
+{% set url = "https://gforge.inria.fr/frs/download.php/file/37696/2018-09-04-14-25-00_GUDHI_2.3.0.tar.gz" %}
+{% set sha256 = "7db6cf3324ac36f82fdda580357329e9f38b79abb7c21db6cc7ae1215668c8f2" %}
+
+package:
+  name: gudhi
+  version: {{ version }}
+
+source:
+  url: {{ url }}
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # C++11 only supports vc14
+  skip: true  # [win and py<35]
```

Comment:
The `py` variable sometimes has the wrong type (int vs str) and is difficult to get right.  I recommend changing this selector to `[win and py2k]`, which has the same effect since conda-forge does not support Python <=3.4